src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int side;
+    private int base;
+
+    public Rectangle(int side, int base, String color) {
```

Comment:
Avoid passing raw values into the constructor. It's better to create variables outside and pass them to the constructor for better readability.